### ISSUE TITLE
PvP buggs

### ISSUE THREAD

**Issue by Muahahahaha** [SunwellTracker/issues#3024]
Decription: Deadzone, and loss Rebuff

How it works: Deadzone: As warrior for example. Mage cast's frostbolt and it's flying to me, i can't spell reflect it, i can spell reflect the next spell he cast's.  
Loss rebuff - I buffed my healer with Commanding Shout, i wanna go behind wall and get myself a Battle Shout, so i loss him and change buffs for both of us. 

How it should work: Deadzone- While spell is flying towards me, i can still reflect it even if u turned spell reflect after cast. Same come for shadowmeld, it shouldn't do dmg when using shadowmeld. Loss rebuff - I buffed my healer with Commanding Shout, i go behind wall( or any obstacle that loss's) and use Battle shout, it only changes for me to Battle shout, the other person shouldn't get rebuffed. 

Source (you should point out proofs of your report, please give us some source): All sources can be easily seen on server.

**Comment by Xilentraz** [comment]
Incorrect. Both things are blizzlike and you can't compare shadowmeld to spell reflection.

**Comment by Muahahahaha** [comment]
Okay, tell me why shadowmeld isn't working properly then? Even if spells flies towards me, when i use shadowmeld it shoudln't do dmg

**Comment by Platinumcoated** [comment]
why not? you made the bug report, you're supposed to prove that it isn't blizzlike, that's what the sources are supposed to be, link some article or comments that say how it worked on retail
